swssconfig/swssconfig.cpp

```diff
@@ -0,0 +1,203 @@
+#include <stdlib.h>
+#include <iostream>
+#include <vector>
+#include "logger.h"
+#include <fstream>
+#include "dbconnector.h"
+#include "producertable.h"
+#include "json.hpp"
+
+using namespace std;
+using namespace swss;
+using json = nlohmann::json;
+
+int db_port							= 6379;
+const char* const hostname 			= "localhost";
+const char* const op_name 			= "OP";
+const char* const op_set 			= "SET";
+const char* const op_del 			= "DEL";
+const char* const name_delimiter 	= ":";
+const int el_count = 2;
+
+#define _in_
+#define _out_
+#define _inout_
```

Comment:
I think it's valuable annotation. We should actually have this moved into common code.